### PR TITLE
Responsiveness Form changes

### PR DESCRIPTION
**Pull request by djnunez-aot**
From MA:
- All the forms on the Entity side need to be fully responsive: the form should remain the same width until the screen gets smaller and then be 100% of the screen.

- The color for validation message should be #CE3E39 (206, 62, 57)

- The validation messages should be “Please enter `<label in lower case>`.” 

- And please replace BC Hydro by CGI Mines Inc. in the first paragraph.

- Another super small thing for the next page (project): Can you change “projects” in the text and on the button for “Project(s)” (keep the lowercase in text and capital on the button but put the “s” in parentheses.)

### CODE REVIEW

File: submit-web/src/components/Shared/controlled/ControlledTextField.tsx [hunk 1]
```diff
@@ -44,7 +46,11 @@ const ControlledTextField: FC<IFormInputProps> = ({
             field.onChange(e.target.value);
           }}
           error={!!errors[name]}
-          helperText={String(errors[name]?.message ?? "")}
+          helperText={
```
Comment (jadmnsaad) [inline submit-web/src/components/Shared/controlled/ControlledTextField.tsx]: use FormHelperTextProps instead

File: submit-web/src/routes/_authenticated/registration/create-account.tsx [hunk 2]
```diff
@@ -56,6 +59,9 @@ function CreateAccount() {
   const methods = useForm({
     resolver: yupResolver(createAccountSchema),
     mode: "onBlur",
+    defaultValues: {
+      phone: "(000) 000-0000",
```
Comment (jadmnsaad) [inline submit-web/src/routes/_authenticated/registration/create-account.tsx]: I think we should do something else here, why do we need to set the default value like that, can't we set a placeholder in the textfield?

**Review by jadmnsaad — CHANGES_REQUESTED** [review]
- helperText change
- default value -> placeholder instead

File: submit-web/src/routes/_authenticated/registration/create-account.tsx (hunk 2 above)
Comment (djnunez-aot) [reply submit-web/src/routes/_authenticated/registration/create-account.tsx]: ill do that

File: submit-web/src/components/Shared/controlled/ControlledTextField.tsx [hunk 3]
```diff
@@ -33,6 +34,7 @@ const ControlledTextField: FC<IFormInputProps> = ({
           {...field}
           inputProps={{
             maxLength: maxLength,
+            height: "32px",
```
Comment (jadmnsaad) [inline submit-web/src/components/Shared/controlled/ControlledTextField.tsx]: if we have to do this, please do it at the theme level

**Review by jadmnsaad — APPROVED** [review]

**Pull request merged**